src/components/Navbar.vue

```diff
@@ -36,85 +35,97 @@
         </ul>
       </div>
     </div>
+
     <div class="flex items-right cart-container">
-        <div class="relative">
-          <button @click="toggleCart" class="hover:text-gray-300 focus:outline-none cart-btn">
+      <div class="relative">
+        <button @click="toggleCart" class="hover:text-gray-300 focus:outline-none cart-btn" ref="cartButton">
           <img class="max-h-9" src="@/components/icons/shopping-cart.png" alt="Cart">
-          </button>
-      <div
-          v-if="isCartOpen" 
-          class=" z-50 absolute right-[-45%] top-14 w-[400px]  text-white backdrop-blur-lg bg-black/30">
-
-          <ul
-          v-for="(cartProduct,index) in cartStore.products " 
-          :key="index">
-            <li>
-              <div class="flex items-center">
-                <img class="p-4 max-h-24" :src="cartProduct.image" :alt="cartProduct.title">
-
-                <div class="p-4 grow">
-                  <p class="text-xl">{{ cartProduct.title }}</p>
-                  <p class="text-xl bold">${{ cartProduct.price }}</p>
+          <span v-if="cartStore.cartItemCount()" class="absolute top-0 right-0 bg-red-500 text-white rounded-full w-6 h-6 text-center">
+            {{ cartStore.cartItemCount() }}
+          </span>
+        </button>
+
+        <div v-if="isCartOpen" ref="cartDropdown" class="z-50 absolute right-[-45%] top-14 w-[400px] text-white backdrop-blur-lg bg-black/30">
+          <div class="max-h-[300px] overflow-y-auto">
+            <ul v-for="(cartProduct, index) in cartStore.products" :key="index">
+              <li>
+                <div class="flex items-center">
+                  <img class="p-2 max-h-16" :src="cartProduct.image" :alt="cartProduct.title">
+                  <div class="p-2 grow">
+                    <p class="text-lg">{{ cartProduct.title }}</p>
```

Comment:
Maybe `text-sm` instead? 🤔

Reply:
I'm also getting a lot of warnings in the console, is it similar for you? 
![image](https://github.com/user-attachments/assets/714853bd-6c04-4756-84f9-b851eca32b26)